packages/dev/materials/src/custom/customMaterial.ts

```diff
@@ -185,19 +171,29 @@ export class CustomMaterial extends StandardMaterial {
             );
         }
 
-        this._isCreatedShader = true;
-        this._createdShaderName = name;
-
         return name;
     }
 
     constructor(name: string, scene?: Scene) {
         super(name, scene);
         this.CustomParts = new ShaderSpecialParts();
         this.customShaderNameResolve = this.Builder;
+        const fn_afterBind = this._afterBind.bind(this);

```

Comment:
there is no need to bind here. this._afterBind is already using an arrow function and thus attached to the context in which you have defined it.

Reply:
_afterBind seems to be override like this
```js
protected _afterBind(mesh?: Mesh | undefined, effect?: Nullable<Effect>) {
        if (!effect) {
            return;
        }
        this.AttachAfterBind(mesh, effect);
        try {
            super._afterBind(mesh, effect);
        } catch (e) {}
}
```